src/auro-input.js

```diff
@@ -69,23 +70,16 @@ export default class AuroInput extends BaseInput {
    * @returns {string} returns string based on type
    */
   handleClickShowPassword() {
-    if (this.type === "password") {
-      this.type = "text";
-      this.showPassword = true;
-      this.focus();
```

Comment:
I wouldn't say it's within the scope of this bug but it would be nice if the focus cursor position didn't jump to the front of the input.

Reply:
@jason-capsule42 please create a separate issue and drop the link here.

Reply:
Oh yeah, that's annoying. Didn't notice it because it doesn't happen on Firefox. Can you open a separate issue for this?

Reply:
https://github.com/AlaskaAirlines/auro-input/issues/33